lucetc/src/function_manifest.rs

```diff
@@ -5,21 +5,62 @@ use failure::{Error, ResultExt};
 use lucet_module_data::FunctionSpec;
 use std::io::Cursor;
 use std::mem::size_of;
+use target_lexicon::BinaryFormat;
 
 fn write_relocated_slice(
     obj: &mut Artifact,
     buf: &mut Cursor<Vec<u8>>,
     from: &str,
-    to: &str,
+    to: Option<&str>,
     len: u64,
 ) -> Result<(), Error> {
-    if len > 0 {
-        obj.link(Link {
-            from, // the data at `from` + `at` (eg. manifest_sym)
-            to,   // is a reference to `to`    (eg. fn_name)
-            at: buf.position(),
-        })
-        .context(format!("linking {} into function manifest", to))?;
+    match (to, len) {
+        (Some(to), 0) => {
+            // This is an imported slice of unknown size
+            let absolute_reloc = match obj.target.binary_format {
+                BinaryFormat::Elf => {
+                    faerie::artifact::Reloc::Raw {
+                        reloc: 1, // this is an ELF R_X86_64_64
```

Comment:
Can you import these constants from `goblin`?

Reply:
sure thing, I was kind of hoping we could avoid depending on `goblin` (directly) just for two constants but oh well :)

Reply:
Its a transitive dependency through faerie. I don't think it hurts :)